Dockerfile

```diff
@@ -7,8 +7,6 @@ RUN rm -rf /tmp/*
 # Install Python
 ENV PYTHONUNBUFFERED=1
 RUN apk add --update --no-cache python3 py3-pip && ln -sf python3 /usr/bin/python
-RUN python3 -m ensurepip
-RUN pip3 install --no-cache --upgrade pip setuptools
```

Comment:
Tried building it, but I think it gets stuck at `line 30`: `RUN GLEAN_PYTHON=python GLEAN_PIP=pip npm run build`

Reply:
```
Dockerfile:30
--------------------
  28 |     ENV S3_BUCKET=$S3_BUCKET
  29 |     
  30 | >>> RUN GLEAN_PYTHON=python GLEAN_PIP=pip npm run build
  31 |     
  32 |     ARG SENTRY_RELEASE
--------------------
ERROR: failed to solve: process "/bin/sh -c GLEAN_PYTHON=python GLEAN_PIP=pip npm run build" did not complete successfully: exit code: 1
```

Reply:
@mansaj Hm, this seemed to work in CircleCI (using the SSH debugging feature), is this locally that it's happening for you?

Reply:
For posterity: we're both having different local docker build issues, but it works in CircleCI so landing as-is to unblock shipping.